app/albums/[id]/edit/page.tsx

```
@@ -1,54 +1,207 @@
+	const router = useRouter();
+	const [loading, setLoading] = useState(true);
+	const [albumData, setAlbumData] = useState<Album | undefined>();
+
+	const [coverPhotoUrl, setCoverPhotoUrl] = useState<string | undefined>(
+		albumData?.coverPhotoUrl,
+	);
+
+	if (!uid) return;
+
+	useEffect(() => {
+		const fetchAlbumsData = async () => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: dispatch

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)